packages/mobile/src/components/News.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {View, Text, Image, StyleSheet, ActivityIndicator} from 'react-native';
+
+const News = (props) => {
```

Comment:
jak spodziewasz sie props w komponencie to typuj prosze te props (twórz type lub interface). w tedy a) masz podpowiedzi co do propsów i trudniej o literówke b) informuje cie ide o bledzie gdy przekażesz props którego nie masz w typie lub nie przekażesz props który masz wymagany